### PR TITLE
VEN-1056 | Upload screenshots of failed browser tests

### PR DESCRIPTION
**Pull request by Nizar-Rahme**
## Description :sparkles:
- Add a step for uploading failed screenshots as GH artifacts
- Fix the customers' list browser tests
- Update the pricing page browser tests

### Closes :no_good_woman:
[**VEN-1056**](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1056): Fix browser tests for Admin UI

### CODE REVIEW

**Comment by ahie** [comment]
From the artifacts it looks like it failed to select all from the table :). But did it fail this action randomly, or is the selector actually broken. I restarted to see.

**Comment by Nizar-Rahme** [comment]
@ahie it was not waiting for the data to be fetched on the customers' list page, so it was failing randomly depending on request handling time. 
Now it's fixed in this PR.

**Review by ahie — APPROVED** [review]
👍 fingers crossed there are no more random failures

File: .github/workflows/browser-tests.yml [hunk 1]
```diff
@@ -40,3 +40,8 @@ jobs:
         run: yarn --prefer-offline
       - name: Run TestCafe tests
         run: yarn browser-test:ci
+      - name: Upload screenshots of failed tests
+        uses: actions/upload-artifact@v2
+        if: failure()
+        with:
+          path: screenshots/
```
Comment (TatuArvela) [inline .github/workflows/browser-tests.yml:47]: Should this have `name: screenshots`?

**Review by TatuArvela — APPROVED** [review]
LGTM! 🎖️✨ Left one thought

File: .github/workflows/browser-tests.yml (hunk 1 above)
Comment (Nizar-Rahme) [reply .github/workflows/browser-tests.yml]: Done

**Pull request merged**